src/components/converter/index.js

```diff
@@ -12,14 +14,23 @@ class Converter extends React.Component {
     this.state = {
       LSK: {
         USD: '0',
+        EUR: '0',
       },
       // 0 index is active one
       currencies: ['USD', 'EUR'],
     };
     this.fee = fees.send;
+
     this.updateData();
   }
 
+  componentDidMount() {
+    const localStorageSettings = localJSONStorage.get('settings', {});
+    if (localStorageSettings.currency) {
+      this.selectActive(localStorageSettings.currency);
+    }
+  }
```

Comment:
The settings object is available in redux store, not need to read it from localStorage and store it in this component's state.